addons/repo-proxy/common/src/main/conf/conf.d/repo-proxy.conf

```diff
@@ -0,0 +1,16 @@
+[repo-proxy]
+# enabled: By default, the repo-proxy add-on is disabled.
+#
+#enabled=false
+
+# proxy: define the proxy rule for the repos. A valid rule is like this: proxy.$packageType.$type.$name=$packageType:$type:$name.
+# proxy.maven.hosted.from=maven:remote:to
```

Comment:
Is the per store? In another word, I assume we don't have many repo need the proxy. Otherwise per store proxy will be too much.

Reply:
This will be abandoned in new way of auto created remote way. Will give pr later.